server/lib/subscriptions.js

```diff
@@ -277,24 +277,30 @@ export async function sendThankYouEmail(order, transaction) {
   const relatedCollectives = await order.collective.getRelatedCollectives(3, 0);
   const recommendedCollectives = await getRecommendedCollectives(order.collective, 3);
   const user = order.createdByUser;
-  return emailLib.send(
-    'thankyou',
-    user.email,
-    {
-      order: order.info,
-      transaction: transaction.info,
-      user: user.info,
-      firstPayment: false,
-      collective: order.collective.info,
-      fromCollective: order.fromCollective.minimal,
-      relatedCollectives,
-      recommendedCollectives,
-      config: { host: config.host },
-      interval: order.Subscription.interval,
-      subscriptionsLink: user.generateLoginLink(`/${order.fromCollective.slug}/subscriptions`),
-    },
-    {
-      from: `${order.collective.name} <hello@${order.collective.slug}.opencollective.com>`,
-    },
-  );
+  const notificationIsActive = await models.Notification.isActive('thankyou', order.collective.id, user.id);
```

Comment:
Have you thought about doing that in the email library (server/lib/email.js) directly, so that we have it available for other templates?

Reply:
Not really but I think It's important to do that now to fix the failing unit test in `lib.subscriptions.test.js`

Reply:
Let's do that.